.github/workflows/desmos_std.yml

```diff
@@ -70,5 +83,8 @@ jobs:
       - name: Run cargo fmt
         run: cargo fmt --all -- --check
 
-      - name: Run cargo clippy
-        run: cargo clippy
+      - name: Run cargo clippy - POAP
+        run: cargo clippy -p poap
+
+      - name: Run cargo clippy - POAP manager
+        run: cargo clippy -p poap-manager
```

Comment:
The error happens because of the unused features dependencies checking of `desmos-bindings` if we run `cargo clippy` on workplace for all contracts.

Reply:
The error messages is as follows, showing that unused features in the `mocks` feature:
```bash
error[E0432]: unresolved import `crate::posts::mocks`
  --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_apps.rs:16:20
   |
16 | use crate::posts::{mocks::mock_posts_query_response, msg::PostsMsg};
   |                    ^^^^^ could not find `mocks` in `posts`

error[E0432]: unresolved import `crate::reactions::mocks`
  --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_apps.rs:20:24
   |
20 | use crate::reactions::{mocks::mock_reactions_query_response, msg::ReactionsMsg};
   |                        ^^^^^ could not find `mocks` in `reactions`

error[E0432]: unresolved import `crate::relationships::mocks`
  --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_apps.rs:22:28
   |
22 | use crate::relationships::{mocks::mock_relationships_query_response, msg::RelationshipsMsg};
   |                            ^^^^^ could not find `mocks` in `relationships`

error[E0432]: unresolved import `crate::reports::mocks`
  --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_apps.rs:24:22
   |
24 | use crate::reports::{mocks::mock_reports_query_response, models::ReportTarget, msg::ReportsMsg};
   |                      ^^^^^ could not find `mocks` in `reports`

error[E0432]: unresolved import `crate::posts::mocks`
 --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_queriers.rs:4:19
  |
4 | use crate::posts::mocks::mock_posts_query_response;
  |                   ^^^^^ could not find `mocks` in `posts`

error[E0432]: unresolved import `crate::reactions::mocks`
 --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_queriers.rs:9:23
  |
9 | use crate::reactions::mocks::mock_reactions_query_response;
  |                       ^^^^^ could not find `mocks` in `reactions`

error[E0432]: unresolved import `crate::relationships::mocks`
  --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_queriers.rs:11:27
   |
11 | use crate::relationships::mocks::mock_relationships_query_response;
   |                           ^^^^^ could not find `mocks` in `relationships`

error[E0432]: unresolved import `crate::reports::mocks`
  --> /home/kilem/.cargo/registry/src/github.com-1ecc6299db9ec823/desmos-bindings-1.0.2/src/mocks/mock_queriers.rs:13:21
   |
13 | use crate::reports::mocks::mock_reports_query_response;
   |                     ^^^^^ could not find `mocks` in `reports`
```

Reply:
This can be fixed if we use `#[cfg(not(target_arch = "wasm32"))]` instead of `#[cfg(all(not(target_arch = "wasm32"), ...))]` inside the `mod.rs` files of `desmos-bindings` like `cosmwasm-std`.
I think that we should go this way since can be annoying for another developer figuring out how to fix this error when runs `clippy`.

Reply:
@manu0466 Thanks so much, I didn't realize it is a bug from desmos-bindings.